frontend/src/screens/apps/ShowApp.tsx

```diff
@@ -156,6 +168,39 @@ function AppInternal({ app, refetchApp, capabilities }: AppInternalProps) {
     }
   };
 
+  const handleConvertToSubwallet = async () => {
+    try {
+      const updateAppRequest: UpdateAppRequest = {
+        name: app.name,
```

Comment:
I recommend all `...app` props are copied.

Reply:
Hmm. I'm a bit hesitant with `...app` as the request payload is not necessarily the same and we end up sending extra properties to the backend.

Although I don't like how it is now. In my opinion ideally we only patch the single field (the metadata field) rather than passing all the data. But we would need more changes for that. I would leave it as-is for now

Reply:
I created a separate issue: https://github.com/getAlby/hub/issues/1546